src/components/review/ReviewsList.js

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect, useState } from 'react';
+import axiosInstance from '../../apis/axiosClient';
+import WriteReview from './WriteReview';
+
+const AlbumReviews = ({ albumId }) => {
+  const [reviews, setReviews] = useState([]);
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    // Fetch reviews for the given album
+    async function fetchAlbumReviews() {
+      try {
+        const response = await axiosInstance.get(`/reviews/album/${albumId}`);
+        console.log(response.data.allProductReviews);
+
+        const { allProductReviews } = response.data;
+        setReviews(allProductReviews);
+        setLoading(false);
+      } catch (error) {
+        console.error('Error fetching album reviews:', error);
+        setLoading(false);
+      }
+    }
+
+    fetchAlbumReviews();
+  }, [albumId]);
+
+  return (
+    <div>
+      <h2>Album Reviews</h2>
+      {loading && <p>Loading reviews...</p>}
+
+      {reviews?.length ? (
+        <ul>
+          {reviews.map((review) => (
+            <li key={review._id}>
+              <h3>{review.title}</h3>
+              <p>Rating: {review.rating}</p>
+              <p>{review.comment}</p>
+            </li>
+          ))}
+        </ul>
+      ) : (
+        'No reviews yet'
+      )}
+      <WriteReview/>
```

Comment:
This component is expecting the `albumId` and a `refreshReviews` function in its props but here we're not passing anything in. That makes the Submit Review button fail because it will try to make a POST call to `http://localhost:8000/api/v1/reviews/album/undefined`

<img width="913" alt="Screenshot 2023-11-13 at 10 28 01 AM" src="https://github.com/Code-the-Dream-School/dd-prac-team4-front/assets/9094098/65c69930-ad6c-41ec-a0b5-ee017b8755d2">

Reply:
This component can define a refreshReviews button which when called, reruns the `fetchAlbumReviews` function and resets the `reviews` variable which will cause this component to rerender the list of reviews and any children it has (WriteReview).